SITCOMTN-023.tex

```diff
@@ -49,13 +48,158 @@
 
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
+\section{Introduction}
+The goal of this document is to define the process for an efficient and timely organization of the work required for a successful commissioning of the Vera C. Rubin Observatory. 
+This process includes defining the current SIT-Com priorities along with their coordination both at the SIT-Com and Project level. 
+The global prioritization of activities is defined by three primary use-cases: (i) the level 2 and level 3 milestones described in P6, (ii) the managerial work from the SIT-Com leadership team (SCLT), and (iii) unplanned emergent issues.
+
+This document first lays down the different use cases along with their appropriate processes and the prioritization strategy.
+ From these uses cases, we defined the requirements on the Jira tool that will satisfy our needs, including work flow, ticket types, review process, dashboard. 
+ Finally, we describe how this Jira project interfaces with the other existing projects. 
+
+\section{Work Management Needs}
+\subsection{Processes}
+As mentioned in the introduction, there are three main applications for the work management process. 
+\subsubsection{Milestone Tracking}
+The P6 tool defined two sets of milestones for SITCom, called L2 and L3. Each milestones are reached when the list of associated activities is completed. 
+Both L2 and L3 milestones will be reported on weekly at the SITCom leadership meetings, especially if delays/blockers are encountered, in which case the impact on the schedule/cost/performance might need to be elevated.
+
+\subsubsection{Planned activities not linked to a milestone}
+The tickets types and processes are similar to the milestone tracking use case, except that it would start at the Epics level or the Story level, depending on the nature of the task. 
+
+\subsubsection{Emergent Tasks}
+Finally, the emergent tasks use case is a mechanism for anyone on the commissioning team to enter issues and bug encountered during commissioning activities. 
+They do not need to be linked to an epic or a milestone while being created. 
+The triage will be done by the SCLT to verify and/or define assignees as well as priorities relative to other work. 
+
+\subsection{Prioritization strategy}
+in order to set up the priority accordingly, the activities will need to have a criticality attached to them.  
+The criticality will be set up by the owner of the ticket and discussed within the SCLT if need be.  
```

Comment:
I meant here the owner of the SCLT group. I changed it to coordinator for now... it will need to be changed to the SITCom subsystem lead eventually (SIT, Com or V&V).